src/renderers/shared/fiber/ReactFiberBeginWork.js

```diff
@@ -169,7 +169,7 @@ module.exports = function<T, P, I, C>(config : HostConfig<T, P, I, C>) {
         // it until the next tick.
         workInProgress.child = current.child;
         reuseChildren(workInProgress, workInProgress.child);
-        if (workInProgress.pendingWorkPriority <= priorityLevel) {
+        if (workInProgress.pendingWorkPriority !== NoWork && workInProgress.pendingWorkPriority <= priorityLevel) {
```

Comment:
Smaller number being higher pri is inconsistent in the way I would read that line